InformationScripting/test/scripts/countChanges.py

```diff
@@ -0,0 +1,17 @@
+# script $<"<changes>" | "<countChanges>" | "<heatmap>">$
```

Comment:
I'm not suggesting what you previously had. That was a single command that did it all.

Now I'm just thinking that we need a generic command for the `countChanges` part of the three-part pipeline above. One which is not specific for commits, but where we can take the things to counts/group by arguments.

And you're right, we don't have to do this in C++. In fact, now that I think about it, it's probably nicer to do it in Python, so that we get to show how it can be used for such, more generic, things.

Does this make sense?